nevergrad/optimization/multiobjective/core.py

```diff
@@ -174,6 +174,16 @@ def pareto_front(
         """
         return self._pf.get_front(size, subset, subset_tentatives)
 
+    def get_min_losses(self) -> tp.List[float]:
+        pf = self._pf.get_raw()
+        min_losses = [np.inf] * len(pf[0].value)
+        for point in pf:
+            point_losses = np.array(point.losses)
```

Comment:
arent they array already?

Reply:
Perhaps. Maybe, I got confused but have changed entire function now to the recommended one liner.